src/main/kotlin/org/jitsi/nlj/rtp/codec/vp8/Vp8Packet.kt

```diff
@@ -97,6 +98,10 @@ class Vp8Packet private constructor (
         }
 
     val temporalLayerIndex: Int = Vp8Utils.getTemporalLayerIdOfFrame(this)
+
+    override val layerId: Int
+        get() = if (hasTemporalLayerIndex) RtpLayerDesc.getIndex(0, 0, temporalLayerIndex) else 0
```

Comment:
Hm, could/should we use `super.layerId` in the else clause, so it works as a single default?

Reply:
Sure.